transit_odp/avl/post_publishing_checks/daily/vehicle_journey_finder.py

```diff
@@ -446,7 +446,9 @@ def match_vehicle_activity_to_vehicle_journey(
 
         txc_xml = self.get_corresponding_timetable_xml_files(matching_txc_file_attrs)
 
-        if not self.filter_by_operating_period(activity_date, txc_xml, result):
+        if not self.filter_by_operating_period(
+            activity.recorded_at_time, txc_xml, result
```

Comment:
I don't think this will work, as `filter_by_operating_period` is expecting a `date` object, not a `datetime` object. (The type hints in the signature tell you this.)
When this function is called with a `datetime` object, it will compare that to `date` objects, raising an exception I think.
You need to use `activity.recorded_at_time.date()` I think.

Reply:
My bad. changed it now..